src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String DATE = "dd.MM.yyyy";
+    private static final String REPORT = "Report for period ";
+    private static final String DASH = " - ";
+    private static final String SPACE = " ";
+    private static final String NEW_LINE = "\n";
+    private static final int SECOND_ELEMENT = 1;
+    private static final int THIRD_ELEMENT = 2;
+    private static final int FOURTH_ELEMENT = 3;
+    private static final int ZERO_ELEMENT = 0;
```

Comment:
constant naming is an opportunity to provide more context, which you are missing here. Name them HOURLY_RATE_INDEX etc, this way it will be more readable to the person who sees the code for the first time